package main

func check(e error) {
	panic(e)
}
